test/e2e/install_test.go

```diff
@@ -27,62 +28,62 @@ var _ = Describe("Operator Install", func() {
 		operatorName string
 		operator     *operatorv1alpha1.Operator
 	)
+	When("An operator is installed from an operator catalog", func() {
+		BeforeEach(func() {
```

Comment:
I think we need an `AfterEach` to cleanup this catalog that we create? Probably should move the operator delete to the `AfterEach` as well to mirror where it gets created.